ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,71 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as **Ready** in the CMS Design System.
+
+## Documentation is complete
+
+**What to check:**
+- [ ] Description of component or pattern
+- [ ] HTML code snippet example(s)
+- [ ] React code snippet example(s)
+- [ ] All react properties are explained
+- [ ] Guidance on when to use this item
+- [ ] Guidance on when not to use this item or consider alternatives
+- [ ] Accessibility considerations
+
+## Validate HTML
+If you are using Google Chrome or Firefox consider getting the [axe plugin](https://www.deque.com/axe) for validation.
+**What to check:**
+- If testing on W3.org
+  - [ ] Go to [validator.w3.org](https://validator.w3.org/#validate_by_input) paste your code in and make sure it validates without error.
+- If testing with axe plugin
+  - [ ] Run the ax plugin on the component or pattern. You should not receive any violations. Some best practices errors such as `Means to skip repeated blocks` of content, or `No main section` can be ignored as they do not apply.
+
+## Screen reader testing
+
+**What to check:**
+- [ ] Windows: IE11 + JAWS
+- [ ] Windows: NVDA + Firefox
+- [ ] MacOS: Safari + VoiceOver
+
```

Comment:
☝️ What else do we need to say here? I've got a doc I started that explains how to set up the first 2. https://github.com/adhocteam/a11y-snippets/blob/7c09d1c8e258b577d8d2b3b1db416d15ec914a49/worksheets/how-to-set-up-screen-readers.md

Is there some information in that a11y repo that explains how to use MacOS VoiceOver?

Reply:
@line47 I've linked to this WebAIM article previously for VoiceOver: https://webaim.org/articles/voiceover/ and like it a lot